data/oauth_client_test.go

```diff
@@ -0,0 +1,137 @@
+package data
+
+import (
+	"database/sql"
+	"github.com/pborman/uuid"
+	"testing"
+)
+
+const (
+	uuidClientGin = "8b14d6bb-cae7-4163-bbd1-f3be46e43e31"
+)
+
+func TestListOAuthClients(t *testing.T) {
+	defer func() {
+		if r := recover(); r != nil {
+			t.Error(r)
+		}
+	}()
```

Comment:
`TestMain` does not really help in this case because it would not allow recover from single tests. But I added a `failOnPanic` function which helps to reduces the redundancy a bit.